src/components/DashboardEditor/editorUtils.jsx

```diff
@@ -300,36 +303,58 @@ const renderListCard = (cardJson, commonProps) => (
 );
 
 /**
- * Returns a Card component for preview in the dashboard
- * @param {Object} cardData, the JSON configuration of the card
- * @param {Function} onSelectCard, callback when card is selected for editing
- * @param {Function} onDuplicateCard, callback when card clone button is clicked
- * @param {Function} onRemoveCard, callback when card delete button is clicked
+ * @param {Object} cardJson
+ * @param {Object} commonProps
  * @returns {Node}
  */
-export const getCardPreview = (
-  cardData,
-  onSelectCard,
-  onDuplicateCard,
-  onRemoveCard
-) => {
-  const commonProps = {
-    key: cardData.id,
-    tooltip: cardData.description,
-    availableActions: { clone: true, delete: true },
-    onCardAction: (id, actionId) => {
-      if (actionId === CARD_ACTIONS.CLONE_CARD) {
-        onDuplicateCard(id);
-      }
-      if (actionId === CARD_ACTIONS.DELETE_CARD) {
-        onRemoveCard(id);
+const renderCustomCard = (cardJson, commonProps) => {
+  return (
+    <Card
+      hideHeader={isNil(cardJson.title)}
+      // need to omit the content because its getting passed content to be rendered, which should not
+      // get attached to the card wrapper
+      {...omit(cardJson, 'content')}
+      {...commonProps}>
+      {
+        // If content is a function, this is a react component
+        typeof cardJson.content === 'function' ? (
+          <cardJson.content />
+        ) : (
+          cardJson.content
+        )
       }
-    },
-    tabIndex: 0,
-    onFocus: () => onSelectCard(cardData.id),
-    className: `${baseClassName}--preview__card`,
-  };
+    </Card>
+  );
+};
 
+/**
+ * Selects the card if the key is 'enter'
+ * @param {Event} evt
+ * @param {Function} onSelectCard
+ * @param {string} id
+ */
+export const handleKeyDown = (evt, onSelectCard, id) => {
+  if (evt.key === 'Enter') {
+    onSelectCard(id);
+  }
+};
```

Comment:
I think the cards are most closely aligned with effectively being 'toggle buttons' since they modify what's shown in the editor panel to the right. 

The selection should happen on <kbd>space</kbd> in addition to <kbd>enter</kbd>. [Related doc from WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-3), I had to look it up 😅 
```suggestion
/**
 * Selects the card if the key is 'enter' or 'space'
 * @param {Event} evt
 * @param {Function} onSelectCard
 * @param {string} id
 */
export const handleKeyDown = (evt, onSelectCard, id) => {
  if (evt.key === 'Enter' || evt.key === 'Space') {
    onSelectCard(id);
  }
};
```

Reply:
Fixed